app/data/jsonSchemaForm/projectEmissionIntensitySchema.ts

```diff
@@ -21,7 +21,10 @@ export const emissionIntensityReportSchema = {
     emissionFunctionalUnit: {
       title: "Functional Unit",
       type: "string",
-      default: undefined,
```

Comment:
default should be "tCO2e"

Reply:
Unfortunately setting the default value here will cause a bug when filling out the form. clearing the field will generate the default value again. in this case I've added the default value to the `add_emission_intensity_report_to_revision` SQL mutation.